### PR TITLE
add deployment to staging

### PR DESCRIPTION
**Pull request by praswicaksono**

### CODE REVIEW

File: .github/workflows/staging_release.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+name: Deploy To Staging Server
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  build:
+    name: Webapp Prod Release
```
Comment (cwagner22) [inline .github/workflows/staging_release.yml:9]: Staging Release

File: .github/workflows/staging_release.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      - name: Test
+        env:
+          GATSBY_API_TEST_USER: ${{ secrets.GATSBY_API_TEST_USER }}
+          GATSBY_API_TEST_PASS: ${{ secrets.GATSBY_API_TEST_PASS }}
+        #run: npm test -- "tradeApiClient"
+        run: echo "skipping tests"
+      - name: Install submodules
+        run: git submodule update --init
+      - name: Build
+        run: npm run build:prod
```
Comment (cwagner22) [inline .github/workflows/staging_release.yml:38]: I think we need to add a new command, i.e. `build:staging`, to point to the correct staging api url for example. (https://staging.zignaly.com/api ?) I can do it tomorrow if needed.

**Comment by cwagner22** [comment]
@praswicaksono Maybe I'll need to tweak the semantic-release plugin we're using to generate releases (https://github.com/zignaly-open/zignaly-webapp/releases) to make it work with the staging workflow.

File: .github/workflows/staging_release.yml (hunk 2 above)
Comment (praswicaksono) [reply .github/workflows/staging_release.yml:38]: yes we need configure the valid url for staging

**Pull request merged**